### PR TITLE
Make persistence optional

### LINKED ISSUE #25: Make persistence optional
Add support for a boolean in the configuration file that can enable or disable the persistence mechanism.

### PR DESCRIPTION
**Pull request by aperrin66**
Closes #25 
Add a parameter in the configuration file that enables to disable the persistence mechanism.

### CODE REVIEW

File: geospaas_harvesting/harvest.py [hunk 1]
```diff
@@ -227,7 +238,12 @@ def main():
                             or not previous_result):
                         #Start a new process
                         results[harvester_name] = pool.apply_async(
-                            launch_harvest, (harvester_name, harvester_config))
+                            launch_harvest, (
+                                harvester_name,
+                                harvester_config,
+                                config.get('dump_on_interruption', True)
```
Comment (opsdep) [inline geospaas_harvesting/harvest.py:248]: With the True in this line, you set the default value of it as True if there is no declaration of it in the harvest.yml file?

**Review by opsdep — DISMISSED** [review]
It is not relevant to this pr but I am curious that why do we need ZeroDivisionError here is this line:
 https://github.com/nansencenter/django-geo-spaas-harvesting/blob/609aeea9c84a3023fd8011e17354227e12cabbe0/tests/test_daemon_script.py#L300

ZeroDivisionError is related to math and those kind of things,isn't it?

**Comment by aperrin66** [comment]
> It is not relevant to this pr but I am curious that why do we need ZeroDivisionError here is this line:
> 
> https://github.com/nansencenter/django-geo-spaas-harvesting/blob/609aeea9c84a3023fd8011e17354227e12cabbe0/tests/test_daemon_script.py#L300
> 
> ZeroDivisionError is related to math and those kind of things,isn't it?

it's just an arbitrary exception raised by the stub class. I chose this one because it's not used anywhere in the code.

**Review by akorosov — DISMISSED** [review]

File: geospaas_harvesting/harvest.py (hunk 1 above)
Comment (aperrin66) [reply geospaas_harvesting/harvest.py:248]: yes

**Pull request merged**